packages/adapter/test/fixtures/security.ts

```diff
@@ -1,5 +1,3 @@
-import { SecuritySchemeSecret } from '@api3/ois';
+import { ApiCredentials } from '@api3/ois';
 
-export const securitySchemeSecrets: SecuritySchemeSecret[] = [
-  { securitySchemeName: 'myapiApiScheme', value: 'super-secret-key' },
-];
+export const credentials: ApiCredentials = { securityScheme: 'myapiApiScheme', value: 'super-secret-key' };
```

Comment:
While you're updating here, might as well update this to be a function instead to be consistent with other fixtures so we get a new object each time it's used

```ts
buildCredentials(overrides?: Partial<ApiCredentials>): ApiCredentials
```